src/awfy.js

```diff
@@ -438,6 +446,30 @@ Object.entries(SITES).forEach(([siteKey, siteLabel]) => {
   });
 });
 
+Object.entries(SITES).forEach(([siteKey, siteLabel]) => {
```

Comment:
I kept desktop live site different from the recording ones until we have results to show, for easier changes